.github/workflows/a11y.yml

```diff
@@ -0,0 +1,158 @@
+name: A11y Tests
+on:
+  pull_request:
```

Comment:
once we have all the kinks worked out, let's update this to run on a schedule like the`broken-links-check.yml` workflow

Something like:
```
on:
  # Manual trigger
  workflow_dispatch:
  # Schedule; currently once nightly, at 3:20 UTC.
  schedule:
    - cron: 20 3 * * *
```
(I've adjusted the hours from the broken link schedule to offset load on the runners)

Reply:
@alexfinnarn everything LGTM, let's bump this to be scheduled and we can merge